addons-reference.md

```diff
@@ -0,0 +1,58 @@
+This document provides a complete reference on addons in Anbox Cloud.
+
+<h2 id='heading--addon-reference-file-structure'>File structure</h2>
+When creating or updating an addon, the directory containing your addon files **MUST**:
+
+- Contain a file named `manifest.yaml`.
+- Contain a directory named `hooks`.
+    - `hooks` **MUST** contain at least one executable file with a valid hook name (see *Hooks* below)
+
+Other files are bundled with the addon and can be accessed in hooks under `$ADDON_DIR` (see [*Environment variables*](#heading--addon-reference-env-variables)).
+```bash
+cat "$ADDON_DIR"/public_key.pem >> ~/.ssh/authorized_keys
+```
```

Comment:
What is this about?

Reply:
This is to showcase an example of an addon accessing a file that was bundled at its creation, I'll add a few steps to make it more comprehensible